frontend/server/src/DAO/Submissions.php

```diff
@@ -157,4 +157,70 @@ public static function countAcceptedSubmissions(
             [$startTimestamp, $endTimestamp]
         );
     }
+
+    /**
+     * @return list<array{time: int, username: string, school_id: int, school_name: string, alias: string, title: string, language: string, verdict: string, runtime: int, memory: int}>
+     */
+    public static function getLastThirtyDaysSubmissions(
```

Comment:
nit:

```suggestion
    public static function getLatestSubmissions(
```

y pásale como parámetro el intervalo en segundos, con un default de `3600 * 24`.